app/docs/0.10.x/proxy.md

```diff
@@ -0,0 +1,759 @@
+[Configuration Reference] for more details.
+
+### Table of Contents
+
+- [Terminology][proxy-terminology]
+- [Overview][proxy-overview]
+- [Reminder: how to add an API to Kong][proxy-reminder]
+- [Routing capabilities][proxy-routing-capabilities]
+    - [Request Host header][proxy-request-host-header]
+        - [Using wildcard hostnames][proxy-using-wildcard-hostnames]
+        - [The `preserve_host` property][proxy-preserve-host-property]
+    - [Request URI][proxy-request-uri]
+        - [The `strip_uri` property][proxy-strip-uri-property]
+    - [Request HTTP method][proxy-request-http-method]
+- [Routing priorities][proxy-routing-priorities]
+- [Proxying behavior][proxy-proxying-behavior]
+    - [1. Load balancing][proxy-load-balancing]
+    - [2. Plugins execution][proxy-plugins-execution]
+    - [3. Proxying & upstream timeouts][proxy-proxying-upstream-timeouts]
+    - [4. Response][proxy-response]
+- [Configuring a fallback API][proxy-configuring-a-fallback-api]
+- [Configuring SSL for an API][proxy-configuring-ssl-for-an-api]
+    - [The `https_only` property][proxy-the-https-only-property]
+    - [The `http_if_terminated` property][proxy-the-http-if-terminated-property]
+- [Proxy WebSocket traffic][proxy-websocket]
+- [Conclusion][proxy-conclusion]
+
+[proxy-terminology]: #terminology
+[proxy-overview]: #overview
+[proxy-reminder]: #reminder-how-to-add-an-api-to-kong
+[proxy-routing-capabilities]: #routing-capabilities
+[proxy-request-host-header]: #request-host-header
+[proxy-using-wildcard-hostnames]: #using-wildcard-hostnames
+[proxy-preserve-host-property]: #the-preserve_host-property
+[proxy-request-uri]: #request-uri
+[proxy-strip-uri-property]: #the-strip_uri-property
+[proxy-request-http-method]: #request-http-method
+[proxy-routing-priorities]: #routing-priorities
+[proxy-proxying-behavior]: #proxying-behavior
+[proxy-load-balancing]: #1-load-balancing
+[proxy-plugins-execution]: #2-plugins-execution
+[proxy-proxying-upstream-timeouts]: #3-proxying-amp-upstream-timeouts
+[proxy-response]: #proxy-response
+[proxy-configuring-a-fallback-api]: #configuring-a-fallback-api
+[proxy-configuring-ssl-for-an-api]: #configuring-ssl-for-an-api
+[proxy-the-https-only-property]: #the-https_only-property
+[proxy-the-http-if-terminated-property]: #the-http_if_terminated-property
+[proxy-websocket]: #proxy-websocket-traffic
+[proxy-conclusion]: #conclusion
+
+---
+
+### Terminology
+
+In this document, we will refer to various Kong concepts. For the sake of 
+consistency, we will now define terms refering to such concepts:
+
+- `API`: This term refers to the API entity of Kong. You configure your APIs,
+that point to your own Upstream services through the Admin API.
+- `Plugin`: This refers to Kong "plugins", which are pieces of business logic
+that run in proxying lifecycle. Plugins can be configured globally (all 
+incoming traffic), or on a per-API basis through the Admin API.
```

Comment:
Plugins can be configured through the Admin API - either globally (all incoming traffic) or on a per-API and/or per-Consumer basis.